static/src/javascripts/projects/common/modules/lazyload.spec.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import { lazyload } from './lazyload';
+
+jest.mock('lib/fetch-json', () => () =>
+    Promise.resolve({ html: '<span>foo</span>' })
+);
+
+describe('Lazy Load', () => {
+    const container = document.createElement('div');
+
+    it('should lazy load', done =>
```

Comment:
You could save the `done` callback by `return lazyload`, because it returns a promise.